server/repositories/cmsQueries/homePageQuery.js

```diff
@@ -49,10 +50,10 @@ class HomepageQuery {
 
   transformEach(item) {
     const [upperFeatured, lowerFeatured] = item.fieldMojFeaturedTileLarge.map(
-      this.#asTile('tile_large'),
+      this.#asTile(getLargeTile),
```

Comment:
meh.. maybe not worth holding this up but could possible get rid of the `uri` from the file fields as presumably no longer used?